src/include/common/config.h

```diff
@@ -27,14 +27,16 @@ extern std::atomic<bool> enable_logging;
 /** If ENABLE_LOGGING is true, the log should be flushed to disk every LOG_TIMEOUT. */
 extern std::chrono::duration<int64_t> log_timeout;
 
+static constexpr const uint64_t INVALID_INDEX = -1;
 static constexpr int INVALID_PAGE_ID = -1;                                    // invalid page id
 static constexpr int INVALID_TXN_ID = -1;                                     // invalid transaction id
 static constexpr int INVALID_LSN = -1;                                        // invalid log sequence number
 static constexpr int HEADER_PAGE_ID = 0;                                      // the header page id
 static constexpr int PAGE_SIZE = 4096;                                        // size of a data page in byte
-static constexpr int BUFFER_POOL_SIZE = 10;                                   // size of buffer pool
+static constexpr int BUFFER_POOL_SIZE = 64;                                   // size of buffer pool
```

Comment:
changed because tables cannot be generated with the default config (reporting cannot create new page). need investigation later.

Reply:
Yes, I'll revert this and set buffer pool size for the shell.